DS_and_Algorithm/bubblesort.py

```diff
@@ -1,3 +1,15 @@
+# Write a bubble sort for python code 
+def bubble_sort(arr):
+    n = len(arr)
+    # Traverse through all elements in the array
+    for i in range(n):
+        # Last i elements are already in place
+        for j in range(0, n-i-1):
+            # Traverse the array from 0 to n-i-1
+            # Swap if the element found is greater than the next element
+            if arr[j] > arr[j+1]:
+                arr[j], arr[j+1] = arr[j+1], arr[j]
+
 def bubble_sort(arr):
```

Comment:
<div>

<strong>Bito Code Review Agent Run</strong> <a href=https://github.com/Amruta101998/DS-and-Algorithm/pull/11#issuecomment-2162735364>#602dd4</a><span> - 06/12/2024, 11:12 am</span>
<br>

<div><small>🔴</small> High importance</div>

<div id="suggestion">
<div id="issue">
<b>Issue:</b> Name "bubble_sort" already defined on line 2
</div>
<div id="fix">
<b>Fix:</b> Rename the second function to avoid redefinition.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 + def bubble_sort_descending(arr: list[int]) -> None:
 - def bubble_sort(arr):
```
</div>
</details>
</div>
<br>

<div id="suggestion">
<div id="issue">
<b>Issue:</b> Redefinition of unused 'bubble_sort' from line 2
</div>
<div id="fix">
<b>Fix:</b> Rename one of the 'bubble_sort' functions to avoid redefinition.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 13c13
 < def bubble_sort(arr):
 ---
 > def bubble_sort_descending(arr):
```
</div>
</details>
</div>
<br>


</div>